src/DotVVM.Framework/Controls/HtmlGenericControl.cs

```diff
@@ -51,6 +51,11 @@ public HtmlGenericControl(string tagName, bool allowImplicitLifecycleRequirement
         public static DotvvmPropertyGroup CssClassesGroupDescriptor =
             DotvvmPropertyGroup.Register<bool, HtmlGenericControl>("Class-", "CssClasses");
 
+        public VirtualPropertyGroupDictionary<object> StyleProperties => new VirtualPropertyGroupDictionary<object>(this, StylePropertiesGroupDescriptor);
```

Comment:
I would rename it to `CssStyles` - just to make it similar to `CssClasses`.

Reply:
ok, let's be consistent...